src/charm.py

```diff
@@ -202,7 +213,8 @@ def _get_s3_env(self):
             "DISCOURSE_S3_ENDPOINT": self.config.get("s3_endpoint", "s3.amazonaws.com"),
             "DISCOURSE_S3_REGION": self.config["s3_region"],
             "DISCOURSE_S3_SECRET_ACCESS_KEY": self.config["s3_secret_access_key"],
-            "DISCOURSE_USE_S3": True,
+            "DISCOURSE_S3_INSTALL_CORS_RULE": str(self.config.get("s3_install_cors_rule", True)).lower(),
```

Comment:
Can't you use a boolean instead?

Reply:
Env vars are only strings, and at some point I got issue due to that, so decided to force string for all env vars

Reply:
Understood